### DISCUSSION TITLE
PCR chimeras detection and exclusion in SQANTI3?

### DISCUSSION THREAD

**Discussion by sfodel** [ConesaLab/SQANTI3#352] [Q&A] [answered]
Dear all,

First of all thanks a lot for your awesome work so far!

My question/recommendation relates to the problem of chimeric molecule formation during PCR and whether/how this can be tackled by SQANTI so far. I have read, and I think I understand, how a similar problem, namely template switching that can occur during the RT, is handled. But what about chimeric molecules that can be formed during PCR? This is very relevant for the latest library preparation protocols from PacBio, for example (e.g., https://www.pacb.com/wp-content/uploads/Procedure-checklist-Preparing-Kinnex-libraries-using-the-Kinnex-full-length-RNA-kit.pdf) where >20 PCR cycles in total across the various protocol steps are present.

Unlike RT template switching, chimera formation during PCR does not have clear sequence/topology related patterns, so I am not entirely sure if the implemented strategy in SQANTI3 will also remove chimeras effectively or not. My fear is that chimera formation can result in "fully valid" artificial "isoforms", especially if the jump happens within, e.g., a common exon across a set of isoforms. I guess these would then fall into, and inflate, the NIC category.

Any insights on the topic? Maybe I am too paranoid or did not read extensively and this issue is nothing to worry about :)

Thanks a lot,
Stelios

**Comment by lkondratova** [comment, marked as answer]
Hi @sfodel,
Normally, we expect some artifact isoforms in the NIC and NNC categories. To mitigate this, we recommend running the SQANTI3 filter followed by the rescue. There are two types of filtering you can use: rules-based and machine learning (ML)-based. The rules filter will flag the isoforms that do not meet the user-specified or default requirements. For the ML rescue, you can supply your list of true positive (TP) and true negative (TN) isoforms to train the classifier to identify artifacts. If the TN and TP lists are not available, they will be generated automatically from the supplied transcriptome using reliable FSM isoforms as TP and NNC isoforms as TN.

Let us know if you have further questions.
